dev/drivers/scripts/plots/mesoscale/jevs_mesoscale_precip_plots.sh

```diff
@@ -35,7 +35,7 @@ export VERIF_CASE="precip"
 export MODELNAME=${COMPONENT}
 
 # EVS Settings
-export HOMEevs="/lfs/h2/emc/vpppg/noscrub/${USER}/EVS"
```

Comment:
Change HOMEevs back to `/lfs/h2/emc/vpppg/noscrub/${USER}/EVS`

Reply:
Fixed.

Reply:
Was there a commit for this fix? I still see HOMevs set as "/lfs/h2/emc/vpppg/noscrub/${USER}/EVS_mesoscale_v2/EVS"